indico/modules/events/timetable/client/js/Toolbar.module.scss

```diff
@@ -17,6 +17,7 @@
 
   :global(.ui.menu) {
     margin-bottom: 0;
+    font-size: 16px;
```

Comment:
can we use `em`-based sizes instead of pixels? in general we're trying to move away from `px` measurements in indico